docs/advanced/solo-proof-of-stake-voting.md

```diff
@@ -0,0 +1,198 @@
+
+`wget https://github.com/decred/decred-release/releases/download/v1.3.0/{dcrinstall-linux-amd64-v1.3.0,manifest-dcrinstall-v1.3.0.txt,manifest-dcrinstall-v1.3.0.txt.asc}`
+
+3. Verify the manifest.
+
+`gpg --verify manifest-dcrinstall-v1.3.0.txt.asc`
+
+4. Verify the SHA-256 hash in the manifest matches that of the binary.
+
+`sha256sum dcrinstall-linux-amd64-v1.3.0`
+
+5. Make the binary executable.
+
+`chmod +x dcrinstall-linux-amd64-v1.3.0`
+
+6. Run it.
+
+`./dcrinstall-linux-amd64-v1.3.0`
+
+**Important**
+
+When you create your wallet, write the 33 seed words down using a pen and paper on a hard surface, and write out two copies.
+
+* **DO NOT** take a picture of the seed words with your phone’s camera.
+* **DO NOT** save the seed words in a text file on a computer.
+* **DO NOT** email the seed words to yourself for "safe keeping".
+
+The only way a person should be able to get the seed words is by acquiring one of the copies you have written out. You should store one copy offsite in case of fire or theft.
+
+---
+
+Once the wallet is created we can do a few things to make life easier. 
+
+1. Create a `bash` script called `decred.sh` which will start a `tmux` session for each application, start `dcrd`, start `dcrwallet`, prompt us for the password to unlock the wallet, and start `dcrctl`. Also make sure you have `tmux` installed.
+
+`echo "tmux new -d -s dcrd 'dcrd & tmux new -d -s dcrwallet 'dcrwallet --promptpass' & tmux attach -t dcrwallet" > ~/decred.sh"`
+
+2. Add the pathe to the Decred binaries to your `.profile`.
+`"PATH=~/decred:$PATH" >> ~/.profile && source ~/.profile`
+
+3. Now make it executable with `chmod +x ~/decred.sh`
+
+Now you can start `dcrd`, `dcrwallet`, and unlock it using `promptsecret` just by running `./decred.sh`.
+If necessary do `tmux attach -t dcrwallet` to enter the password and then `tmux attach -t dcrctl` to enter commands
+
+The next step will be to start buying tickets manually or using `ticketbuyer` in conjunction with a series of hot wallets you will set up later in this guide.
+
+---
+
+## Ticket Buying
+Now from your cool wallet you can purchase tickets using the following command once your wallet is unlocked:
+
+`dcrctl --wallet purchaseticket default 150 1 DsHotWalletAddressFromVPS 10`
+
+Replacing `DsHotWalletAddressFromVPS` with your hot wallet voting address which you will generate when you set up your VPS for voting and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your VPS instances.
+
+If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf` where `DsHotWalletAddressFromVPS` is once again the address you generated on your voting VPS.
+
+```
+enableticketbuyer=1
+ticketbuyer.votingaddress=DsHotWalletAddressFromVPS
+ticketbuyer.balancetomaintainabsolute=0
+```
+
+---
+
+## Voting
```

Comment:
We have two headers "Voting" - could we differentiate them with "Politeia Voting" and "Proof-of-stake voting" ?